### PR TITLE
Add installation instructions for MacPorts

### PR DESCRIPTION
**Pull request by gpanders**

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -17,10 +17,18 @@ After installing, `clangd --version` should print `clangd version 7.0.0` or late
...
+ 
 Clangd can be installed (along with LLVM) via [Homebrew](https://brew.sh):
 ```
 brew install llvm
 ```
+
+or with [MacPorts](https://www.macports.org/):
+
+```
+sudo port install clang-11
```
Comment (njames93) [inline installation.md:29]: Shouldn't this be clangd-11

File: installation.md [hunk 2]
```diff
@@ -17,10 +17,18 @@ After installing, `clangd --version` should print `clangd version 7.0.0` or late
 
 <details>
 <summary markdown="span">Mac OS X</summary>
+
```
Comment (njames93) [inline installation.md:20]: Any reason for this empty line?

File: installation.md (hunk 1 above)
Comment (gpanders) [reply installation.md:29]: clangd is installed as part of the clang package.

File: installation.md (hunk 2 above)
Comment (gpanders) [reply installation.md:20]: I think (though I could be mistaken) that raw HTML in Markdown is supposed to have a blank line after it. At the very least, it makes the [file preview in GitHub look better](https://github.com/llvm/clangd-www/pull/32/files?short_path=a0ab6b4#diff-a0ab6b4c22ea1ad409f8e2271ae1c1220e49c6a2782afa92ff2462367cbb37f5).

File: installation.md (hunk 1 above)
Comment (sam-mccall) [reply installation.md:29]: It's a bit unfortunate this has a version number in it, but I'm sure people can work that out if our docs get stale.

(Versioning compiler packages makes sense of course, clangd less so but bundling them makes practical sense too, so...)

**Comment by sam-mccall** [comment]
Thanks!

File: installation.md (hunk 1 above)
Comment (gpanders) [reply installation.md:29]: Would it make sense to change it to something like this instead:

````markdown

or with [MacPorts](https://www.macports.org/) using one of the available `clang` ports, e.g.

```
sudo port install clang-11
```

Use `port search clang` to find all available versions.
````

In this case `clang-11` is just an example, so even when `clang-12` or `clang-13` are released, the docs won't appear stale.

**Pull request merged**